app/javascript/lib/scp-api.jsx

```diff
@@ -243,6 +243,19 @@ export function setupRenewalForReadOnlyToken(studyAccession) {
   }, renewalTime)
 }
 
+
+/**
+ * Renew userAccessToken
+ * @param {String} studyAccession Study accession, e.g. SCP123
+ */
+export async function renewUserAccessToken(studyAccession) {
```

Comment:
Thinking out loud: Right now, there's always a study being used in context with this request. But what if the user is on the home page and their session is about to expire? It impacts search/download requests, so it's definitely a valid context. However, the workaround of refreshing the page works fine as they won't lose state, so I don't consider it blocking. It is something we should consider moving forward, though - the renewUserAccessToken method should likely be adapted to handle this case.